ui/packages/diagnosis_report/scripts/watch.js

```diff
@@ -0,0 +1,53 @@
+// https://gist.github.com/int128/e0cdec598c5b3db728ff35758abdbafd
+process.env.NODE_ENV = 'development'
```

Comment:
Now there are 3 build systems..1 is CRA, 1 is rollup, 1 is this watch file. It is horrible for such a simple project that there is actually only one App targeting at users.

Reply:
yes it is! it is horrible for me as well... do you have any good ideas? but I think the diagnosis report is an independent app with the dashboard ui.

Reply:
if creat-react-app can add one more option for react-scripts to enable webpack watch mode, we don't need this file at all, the discussion is here: https://github.com/facebookincubator/create-react-app/issues/1070 , I have viewed all comments and tried some methods someone supplied, and this should be the easiest and works one.

I also tried to change the webpack config in the config-overrides.js manually, change `config.watch = true` but unfortunately, it doesn't work ...